src/nat/eval/runners/red_teaming_runner/runner.py

```
@@ -0,0 +1,848 @@
+            logger.info("Generating workflow config for scenario: %s", scenario_id)
+
+            # Deep copy the base workflow config
+            base_workflow_config_dict = self.base_workflow_config.model_dump(mode='python', exclude_unset=False)
+
+            # Add only the LLMs that are actually used by scenarios
+            for llm_name in required_llm_names:
+                if llm_name not in self.config.llms:
+                    raise ValueError(
+                        f"Scenario '{scenario_id}' references LLM '{llm_name}' "
+                        f"but it's not defined in the llms dict"
+                    )
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)